src/test_workflow/test_cluster.py

```diff
@@ -48,6 +48,8 @@ def __init__(
         self.save_logs = save_logs
         self.all_services = []
         self.termination_result = None
+        self.start_stdout = ""
```

Comment:
Semantically this is incorrect, the default stdout/stderr is `None`. Set it to that?

Reply:
Hi, @dblock , thanks for the asking..because later at L106-107, I want to concatenate two strings, so, I